app/src/main/kotlin/org/fossify/keyboard/extensions/Context.kt

```diff
@@ -207,6 +207,7 @@ fun Context.getKeyboardLanguagesRadioItems(): ArrayList<RadioItem> {
 fun Context.getKeyboardLanguageText(language: Int): String {
     return when (language) {
         LANGUAGE_ARABIC -> getString(R.string.translation_arabic)
+        LANGUAGE_BELARUSIAN_CYRL -> getString(R.string.translation_belarusian)
```

Comment:
This string doesn't exist. You should add it in Commons in https://github.com/FossifyOrg/Commons/blob/master/commons/src/main/res/values/strings.xml. When creating a pull request in Commons, please mention that it's for this one.

Reply:
Done: https://github.com/FossifyOrg/Commons/pull/60

Thank you for your instructions!